runJupyterDaskOnSLURM.py

```diff
@@ -379,24 +379,40 @@ def forward_port_and_launch_local(conn,forwardconfig):
         else:
             print('Launching webbrowser failed')
 
-
 def main():
     """
     Run through steps to launch JupyterDaskOnSLURM instance on remote platform
     """
-
     # parse command line arguments
     args = parse_cla()
     # retrieve or set config
     config_inputs, platform_name = get_config(args)
-    # submit batch job with scheduler
-    outfilename = ssh_remote_executor(config_inputs, submit_scheduler, args, platform_name)
-    forwardconfig = ssh_remote_executor(config_inputs, check_and_retrieve_SLURM_info, outfilename, args)
-    if forwardconfig is None:
-        print('submission appears to have failed.')
+    # Check and install on remote as needed
+    user_install = input('Are required components already installed on remote host? (Y/n): ')
+    if user_install == 'Y':
+        install = True
     else:
-        _ = ssh_remote_executor(config_inputs,forward_port_and_launch_local, forwardconfig)
-
+        install = installJDOnSLURM.install_JD(config_inputs, platform_name, envfile = 'environment.yaml')
+
+    user_uninstall = input('Do you want to uninstall all components on remote host? (Y/n): ')
```

Comment:
I had an issue here, as I typed "y" (lower-case) and this was not recognised as a yes. Also, the "(Y/n)" prompt suggests "Y" to be the default, but actually there is no default. Something like the following could fix these issues?
```suggestion
    user_uninstall = (input('Do you want to uninstall all components on remote host? (Y/n): ') or "Y").upper()
```
We might then also have to catch the case in which one types in something different from "y" or "n".